app/src/main/java/com/google/maps/android/compose/MarkerClusteringActivity.kt

```diff
@@ -171,7 +173,19 @@ private fun CustomUiClustering(items: List<MyItem>) {
 @OptIn(MapsComposeExperimentalApi::class)
 @Composable
 fun CustomRendererClustering(items: List<MyItem>) {
+    val configuration = LocalConfiguration.current
+    val screenHeight = configuration.screenHeightDp.dp
+    val screenWidth = configuration.screenWidthDp.dp
     val clusterManager = rememberClusterManager<MyItem>()
+
+    // Here the clusterManager is being customized with a NonHierarchicalViewBasedAlgorithm.
+    // This speeds up by a factor the rendering of items on the screen.
+    clusterManager?.setAlgorithm(
+        NonHierarchicalViewBasedAlgorithm(
+            screenHeight.value.toInt(),
+            screenWidth.value.toInt()
+        )
```

Comment:
Hi @kikoso!

I was checking the PR to implement the algorithm in my app, and I noticed that the constructor params of the showcase are reversed. 🙂

`public NonHierarchicalViewBasedAlgorithm(int screenWidth, int screenHeight)`